org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/main/Generate.java

```diff
@@ -393,20 +393,14 @@ public void registerResourceFactories(ResourceSet resourceSet) {
         
         /*
          * TODO If you need additional resource factories registrations, you can register them here. the following line
-         * (in comment) is an example of the resource factory registration.
```

Comment:
@berezovskyi please don't ask. These comments are generated by Acceleo everytime I update the project. It's a pain to keep reverting. I am not sure what we get out of changing them?